arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts

```diff
@@ -208,19 +208,22 @@
 		/*pinctrl-0 = <&pinctrl_pi4io16_A>;*/
 		#gpio-cells = <2>;
 		status = "okay";
+		/* below tags for gpio pins are updated to sapphire 2.3.4.2 */
```

Comment:
below tags for gpio pins are updated *with respect* to sapphire 2.3.4.2 *schematic*

Reply:
will update in future